engine/baml-runtime/src/internal/llm_client/orchestrator/mod.rs

```diff
@@ -7,8 +7,11 @@ use baml_types::BamlValue;
 use internal_baml_core::ir::repr::IntermediateRepr;
 use internal_baml_jinja::RenderedChatMessage;
 use internal_baml_jinja::RenderedPrompt;
+use js_sys;
+use serde::Serialize;
 use std::{collections::HashMap, sync::Arc};
-use web_time::Duration;
+use wasm_bindgen::JsValue;
```

Comment:
too much leakage of JsValue into runtimes (python, ruby, etc. This is a WASM only feature and all things related to wasm_bindgen / JSValue should be behind a cfg for wasm target.

Not doing this can lead to hard to detect bugs in runtimes when mixing and matching WASM with py03, typescript, etc.

Reply:
Sorry forgot about that.